johncena.dart

```diff
@@ -0,0 +1,5 @@
+void main() {
+  while(true){
+    print('');
```

Comment:
can you optimise this?
this doesn't look optimised

Reply:
I'll merge it for now, see if you can make it faster. It's no fun to just send pr